connectors/src/connectors/github/temporal/activities.ts

```diff
@@ -268,21 +271,20 @@ export async function githubUpsertDiscussionActivity(
     throw new Error("Connector not found");
   }
 
-  const existingDiscussionInDb = await GithubIssue.findOne({
+  const existingDiscussionInDb = await GithubDiscussion.findOne({
```

Comment:
Can you use `GithubDiscussion.upsert()` instead of `GithubDiscussion.findOne/create`? That way you are guaranteed to never crash on that.

Reply:
thanks